### DISCUSSION TITLE
HTML Report doesnt look ok in my github Repository.

### DISCUSSION THREAD

**Discussion by EmmanuelHape** [IBAHCM/RPiR#167] [Archive] [answered]
@richardreeve @soniamitchell

Hi Richard & Sonia

I have uploaded the scripts in my repository for practical _0202 but the HTML report doesn't read fine in the repository.
The report on my computer is fine, I have tried to pull it out it doesn't work. Any help??

**Comment by soniamitchell** [comment, marked as answer]
Do you mean [this](https://github.com/IBAHCM/emmanuelhape_series02/blob/main/0202_run-SIS.html)? If so, it does read fine, don't worry. That's what html code looks like before it's interpreted by your browser.

**Reply by soniamitchell** [reply]
I would however recommend not pushing `.html` files to GitHub, since we can compile them locally from the `.Rmd` file 👍

**Reply by EmmanuelHape** [reply]
Yes, that is what I meant. Thanks then.
